components/feature/session/src/main/java/mozilla/components/feature/session/PictureInPictureFeature.kt

```diff
@@ -32,7 +34,9 @@ class PictureInPictureFeature(
             return false
         }
 
-        val fullScreenMode = sessionManager.selectedSession?.fullScreenMode ?: false
+        val customTabSession = customTabSessionId?.let { sessionManager.findSessionById(it) }
+        val session = customTabSession ?: sessionManager.selectedSession
+        val fullScreenMode = session?.fullScreenMode ?: false
```

Comment:
We have [`SessionManager.runWithSessionIdOrSelected`](https://searchfox.org/mozilla-mobile/rev/3ff3538a0d71de69a42ad7be0a16dfc6ca9cc483/android-components/components/browser/session/src/main/java/mozilla/components/browser/session/SessionManager.kt#474) that might work here.

Reply:
Ah that's what I was looking for! Thanks!